x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

```diff
@@ -350,6 +354,7 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
 
       it(`shows the visualize button`, async () => {
         await PageObjects.common.navigateToApp('discover');
+        await PageObjects.common.waitForTopNavToBeVisible();
```

Comment:
**question** Should `navigateToApp`'s default behavior simply call `waitForTopNavToBeVisible`?

Reply:
No, nat all apps have a top nav. Discover does though and it's a good way of checking whether the app is loaded (without waiting for the data table to load).